sample/src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,292 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React from 'react';
+import {
+  Image,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import {StackNavigationProp} from '@react-navigation/stack';
+
+import * as Sentry from '@sentry/react-native';
+import {getTestProps} from '../../utils/getTestProps';
+
+interface Props {
+  navigation: StackNavigationProp<any, 'HomeScreen'>;
+}
+
+const HomeScreen = (props: Props) => {
+  const currentDSN = Sentry.getCurrentHub().getClient().getOptions().dsn;
+  const ourDSN =
+    'https://d870ad989e7046a8b9715a57f59b23b5@o447951.ingest.sentry.io/5428561';
```

Comment:
should we not use `SENTRY_INTERNAL_DSN`?

Reply:
Yes we should